packages/bot-web-ui/src/pages/server-side-bot/summary/summary.tsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Button, Icon, Money, Text } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import { Localize, localize } from '@deriv/translations';
+import { useDBotStore } from 'Stores/useDBotStore';
+
+const STATUS = Object.freeze({
+    open: {
+        text: localize('Running'),
+        icon: 'IcClockOutline',
+        icon_size: 14,
+        class_name: 'ssb-summary__item__header__result--progress',
+        color: '',
+    },
+    won: {
+        text: localize('Won'),
+        icon: 'IcServerBotProfit',
+        icon_size: 20,
+        class_name: 'ssb-summary__item__header__result--won',
+        color: 'green',
+    },
+    lost: {
+        text: localize('Lost'),
```

Comment:
```suggestion
import { Localize } from '@deriv/translations';
import { useDBotStore } from 'Stores/useDBotStore';

const STATUS = Object.freeze({
    open: {
        text: <Localize i18n_default_text='Running' />,
        icon: 'IcClockOutline',
        icon_size: 14,
        class_name: 'ssb-summary__item__header__result--progress',
        color: '',
    },
    won: {
        text: <Localize i18n_default_text='Won' />,
        icon: 'IcServerBotProfit',
        icon_size: 20,
        class_name: 'ssb-summary__item__header__result--won',
        color: 'green',
    },
    lost: {
        text: <Localize i18n_default_text='Lost' />,
```

Reply:
I think this is just matter personal choice, doesn't make much of difference.

@maryia-deriv ?